resources/hooks/local/pre-push

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+remote="$1"
+
+while read local_ref local_sha remote_ref remote_sha
+do
+    # Get the branch name
+    local_ref=$(basename $local_ref)
+    
+    # Fetch the GIT diff and format it as command input:
+    DIFF=$(git diff --name-only "$remote"/"$local_ref"..HEAD --oneline)
+        
+    # Run GrumPHP
+    (cd "./" && printf "%s\n" "${DIFF}" | exec 'vendor/phpro/grumphp/bin/grumphp' 'git:pre-push')
```

Comment:
This line is inside a while which is executed for every line in the STDIN.
More info:

```
while read local_ref local_sha remote_ref remote_sha
```

Every input line is a commit which is pushed to the remote. So when there are many commits in one push, the grumphp executable is executed multiple times.